### PR TITLE
Add default timeout and skip flaky tests

### PR DESCRIPTION
**Pull request by bzsolt**
The default timeout is 240 seconds now.

IoT.js-DCO-1.0-Signed-off-by: Zsolt Borbély zsborbely.u-szeged@partner.samsung.com

### CODE REVIEW

File: tools/test_runner.js [hunk 1]
```diff
@@ -18,6 +18,8 @@ var console_wrapper = require('common_js/module/console');
 var builtin_modules =
   Object.keys(process.native_sources).concat(Object.keys(process.binding));
 
+var default_timeout = 240; // (in seconds)
```
Comment (zherczeg) [inline tools/test_runner.js]: That is a pretty long timeout. A bad patch (where all tests are timed out) could make the test system run kind of forever.

Comment (bzsolt) [inline tools/test_runner.js]: Yes, it seems a pretty big timeout, but a debug binary + valgrind combination can produce 100+ sec runtimes on Travis CI (moreover tomorrow I've seen a 169 sec runtime), so this value based on this corner-cases, but ofc we can adjust it. Now there is no limitation for the time of a testrun unless the test specifies a certain timeout in `testsets.json`.

Comment (kolipka) [inline tools/test_runner.js]: @bzsolt Would it be possible to have it as a command line parameter? 240 is fine for me, but if not I could override it.

Comment (bzsolt) [inline tools/test_runner.js]: @kolipka great idea, I've added an additional option to `check_test.js`, please check it.

File: tools/common_js/option_parser.js [hunk 2]
```diff
@@ -60,6 +62,11 @@ OptionParser.prototype.parse = function() {
     var val = arg_val[1];
     var found = false;
 
+    if (arg == "default-timeout") {
```
Comment (kolipka) [inline tools/common_js/option_parser.js]: == is error prone, please use ===

File: tools/test_runner.js (hunk 1 above)
Comment (kolipka) [inline tools/test_runner.js]: @bzsolt looks awesome, thanks!

**Comment by yichoi** [comment]
LGTM
